src/portable/mm32/dcd_mm32f327x_otg.c

```diff
@@ -0,0 +1,462 @@
+////////////////////////////////////////////////////////////////////////////////
+/// @file     dcd_mm32f327x_otg.c
+/// @author   SE TEAM
+/// @brief    tinyusb driver.
+////////////////////////////////////////////////////////////////////////////////
+/// @attention
+///
+/// THE EXISTING FIRMWARE IS ONLY FOR REFERENCE, WHICH IS DESIGNED TO PROVIDE
+/// CUSTOMERS WITH CODING INFORMATION ABOUT THEIR PRODUCTS SO THEY CAN SAVE
+/// TIME. THEREFORE, MINDMOTION SHALL NOT BE LIABLE FOR ANY DIRECT, INDIRECT OR
+/// CONSEQUENTIAL DAMAGES ABOUT ANY CLAIMS ARISING OUT OF THE CONTENT OF SUCH
+/// HARDWARE AND/OR THE USE OF THE CODING INFORMATION CONTAINED HEREIN IN
+/// CONNECTION WITH PRODUCTS MADE BY CUSTOMERS.
+///
+/// <H2><CENTER>&COPY; COPYRIGHT MINDMOTION </CENTER></H2>
```

Comment:
copyright for the drivers file that is within `/src` must be MIT license to be compatible with the rest of the project. Please consult your company and put the MIT license here if you are willing to move forward with the PR.

Reply:
Note: only this file need to be MIT license. Other driver in your sdk does not need to change at all